npm/css-src/sass/_success_summary.scss

```diff
@@ -0,0 +1,106 @@
+// We have adapted this component https://design-system.service.gov.uk/components/error-summary/ from the GOV UK design system.
+.success-summary {
+  border: 5px solid #00703c;
+  color: #0b0c0c;
+  margin-bottom: 30px;
+  padding: 15px; }
+@media print {
+  .success-summary {
+    color: #000; } }
+@media (min-width: 40.0625em) {
+  .success-summary {
+    padding: 20px; } }
+@media (min-width: 40.0625em) {
+  .success-summary {
+    margin-bottom: 50px; } }
+
+.success-summary:focus {
+  outline: 3px solid #fd0; }
+
+.success-summary__title {
+  font-family: Arial, sans-serif;
+  font-size: 18px;
+  font-size: 1.125rem;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  font-weight: 700;
+  line-height: 1.11111;
+  margin-bottom: 15px;
+  margin-top: 0; }
+@media print {
+  .success-summary__title {
+    font-family: sans-serif; } }
+@media (min-width: 40.0625em) {
+  .success-summary__title {
+    font-size: 24px;
+    font-size: 1.5rem;
+    line-height: 1.25; } }
+@media print {
+  .success-summary__title {
+    font-size: 18pt;
+    line-height: 1.15; } }
+@media (min-width: 40.0625em) {
+  .success-summary__title {
+    margin-bottom: 20px; } }
+
+.success-summary__body {
+  font-family: Arial, sans-serif;
+  font-size: 16px;
+  font-size: 1rem;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  font-weight: 400;
```

Comment:
Do a lot of these things inherit from other text styles in some way? I'm just surprised by how much we've had to copy from the design system, and I'm wondering if there's a recommended way to do it?